nevergrad/optimization/optimizerlib.py

```diff
@@ -1815,6 +1816,154 @@ def __init__(
+
+        self._buffer: tp.List[float] = []
+        self._newX: tp.List[float] = []
+        self._losses: tp.List[float] = []
+
+        # Setting DoE size as a percentage of the total budget if prop_doe_factor is not None
+        init_budget = cfg.init_budget
+        if cfg.prop_doe_factor and budget is not None:
+            init_budget = round(cfg.prop_doe_factor * budget) if budget >= 10 else 5
+
+        if cfg.pca:
+            from bayes_optim.extension import PCABO as PcaBO
+
+            self._alg = PcaBO(
+                search_space=space,
+                obj_fun=None,  # Assuming that this is not used :-)
+                DoE_size=init_budget if init_budget is not None else 5,
+                max_FEs=budget,
+                verbose=True,
+                n_point=1,  # We start with a sequential procedure, maybe we'll extend in a second moment
+                n_components=cfg.n_components,
+                acquisition_optimization={"optimizer": "BFGS"},
+            )
+        else:
+            from bayes_optim import BO as BayesOptimBO
+
+            # hyperparameters of the GPR model
+            thetaL = 1e-10 * (ub - lb) * np.ones(self.dimension)
+            thetaU = 10 * (ub - lb) * np.ones(self.dimension)
+            model = GaussianProcess(thetaL=thetaL, thetaU=thetaU)  # create the GPR model
+
+            self._alg = BayesOptimBO(
+                search_space=space,
+                obj_fun=None,  # Assuming that this is not used :-)
+                model=model,
+                DoE_size=init_budget if init_budget is not None else 5,
+                max_FEs=budget,
+                verbose=True,
+            )
+
+    def _internal_ask_candidate(self) -> p.Parameter:
+        if not self._buffer:
+            self._newX = []
+            self._losses = []
+            candidate = self._alg.ask()
+            if not isinstance(candidate, list):
+                candidate = candidate.tolist()
+            self._buffer = candidate
+        x_probe = self._buffer.pop()
+        data = np.tan(np.array(x_probe, copy=False))
+        candidate = self.parametrization.spawn_child().set_standardized_data(data)
+        candidate._meta["x_probe"] = x_probe
+        return candidate
+
+    def _internal_tell_candidate(self, candidate: p.Parameter, loss: tp.FloatLoss) -> None:
+        self._newX.append(candidate._meta["x_probe"])
+        self._losses.append(loss)
+        if not self._buffer:
+            if "x_probe" in candidate._meta:
+                self._alg.tell(self._newX, self._losses)
+            else:
+                data = candidate.get_standardized_data(reference=self.parametrization)
+                # Tell not asked:
+                self._alg.tell(np.arctan(data), loss)
+
+    def _internal_tell_not_asked(self, candidate: p.Parameter, loss: tp.FloatLoss) -> None:
+        raise errors.TellNotAskedNotSupportedError
+
+
+class ParametrizedBayesOptim(base.ConfiguredOptimizer):
```

Comment:
let's call this just `BayesOptim`.
Please also import it in optimization/families.py I think this way it will be automatically documented

Reply:
Why woulf you remove "Parametrized"? We can actually choose a couple of parameters for PCABO: 
```
self.n_components = n_components
self.prop_doe_factor = prop_doe_factor
```